src/connectors/spotify.js

```diff
@@ -25,10 +25,6 @@ Connector.isScrobblingAllowed = () => {
 };
 
 function isMusicPlaying() {
-	if (Connector.getArtist() !== 'Spotify') {
```

Comment:
What does removing this achieve?

Reply:
Many Spotify ads have artist values other than "Spotify", so this condition currently allows those ads to be scrobbled.

Reply:
Where is the part that stops the disallow then? Is there anything in the DOM to highlight this?

Reply:
With this code removed, the remainder of `isMusicPlaying()` stops the disallow when the artist link href is an artist page instead of an ad URL. That URL logic still works well, but the "Spotify" artist name logic doesn't work and prevents the URL logic from being reached.